docs/learn-phragmen.md

```diff
@@ -547,20 +597,20 @@ expensive proposition. Although obtaining an equal distribution is unlikely, the
 distribution, the higher the threshold - and thus the higher the expense - for attackers to gain
 entry to the set.
 
-### High-Level Description
+### Rationale for Reducing Block Computing Time
 
-After running the weighted Phragmén algorithm, a process is run which redistributes the vote amongst
-the elected set. This process will never add or remove an elected candidate from the set. Instead,
-it reduces the variance in the list of backing stake from the voters to the elected candidates.
-Perfect equalization is not always possible, but the algorithm attempts to equalize as much as
-possible.
+Running the Phragmén algorithm is time-consuming, and often cannot be completed within the time
+limits of production of a single block. Waiting for calculation to complete would jeopardize the
+constant block production time of the network. Therefore, as much computation as possible is moved
+to an offchain worker, which validators can work on the problem without impacting block production
+time. By restricting the ability of users to make any modifications in the last 25% of an era, and
+running the selection of validators by nominators as an offchain process, validators have a
+significant amount of time to calculate the new active validator set and allocate the nominators in
+an optimal manner.
 
-These optimizations will not be covered in-depth on this page. For more details, you can view the
-[Rust implementation in Substrate](https://github.com/paritytech/substrate/blob/master/frame/elections-phragmen/src/lib.rs)
-or the `seqPhragménwithpostprocessing` method in the
-[Python reference implementation](https://github.com/w3f/consensus/tree/master/NPoS). If you would
-like to dive even more deeply, you can review the
-[W3F Research Page on Sequential Phragmén Method](https://research.web3.foundation/en/latest/polkadot/NPoS/4.%20Sequential%20Phragm%C3%A9n%E2%80%99s%20method.html).
+There are several further restrictions put in place to limit the complexity of the election and
+payout. As already mentioned, any given nominator can only select up to 16 validators to nominate.
+Conversely, a single validator can have only 64 nominators.
```

Comment:
Should we warn people about checking that the validators they're nominating aren't already saturated or they'd be wasting nominations?

Reply:
Do you know an easy way to do this?  We won't know that a validator is saturated until after the Phragmen election, and you have to nominate well before then.  One could make guesses ahead of time by running offline-Phragmen, I suppose.

I don't think it will be a big deal, at least at first, since the edge-reductions minimization reduces this problem considerably.  On Kusama, for instance, we can have 225 * 64 (14,400) nominating accounts.  I don't THINK we're getting close to that (but I am having trouble getting the kusama-validator-stats script to work right now...)

I can mention it here, though.